src/sagemaker/session.py

```diff
@@ -115,7 +115,7 @@ def _initialize(self, boto_session, sagemaker_client, sagemaker_runtime_client):
     def boto_region_name(self):
         return self._region_name
 
-    def upload_data(self, path, bucket=None, key_prefix='data'):
+    def upload_data(self, path, bucket=None, key_prefix='data', ExtraArgs=None):
```

Comment:
let's use snake case (i.e. `extra_args`) here to match Python convention

Reply:
Changed to extra_args